extensions/positron-python/src/client/positron/interpreterSettings.ts

```diff
@@ -209,3 +210,27 @@ export function printInterpreterDebugInfo(interpreters: PythonEnvironment[]): vo
     traceInfo('================ [END] PYTHON INTERPRETER DEBUG INFO ================');
     traceInfo('=====================================================================');
 }
+
+/**
+ * Retrieves the user's default Python interpreter path from VS Code settings
+ *
+ * @returns The configured Python interpreter path if it exists and is not 'python',
+ *          otherwise returns an empty string
+ */
+export function getUserDefaultInterpreter(scope?: Resource): InspectInterpreterSettingType {
+    const configuration = vscode.workspace.getConfiguration('python', scope);
+    const defaultInterpreterPath: InspectInterpreterSettingType =
+        configuration?.inspect<string>('defaultInterpreterPath') ?? {};
```

Comment:
For the interpreter includes/excludes further up in this file, we `untildify` the path and check if it's absolute (logging if it's not). Should we do that here as well?

Reply:
Added a check if it is absolute, but the [setting does untildify on update](https://github.com/posit-dev/positron/blob/bb26d926ef030ff8d147f048718669b55eeb6ca3/extensions/positron-python/src/client/common/configSettings.ts#L546)